Assets/MRTK/SDK/Features/UX/Prefabs/InteractiveElement/CompressableButtonCube.prefab

```diff
@@ -220,9 +193,9 @@ MonoBehaviour:
         OnTouchUpdated:
           m_PersistentCalls:
             m_Calls: []
-    00000003:
-      type: {class: PressedNearEvents, ns: Microsoft.MixedReality.Toolkit.UI.Interaction,
-        asm: Microsoft.MixedReality.Toolkit.SDK}
+    00000002:
+      type: {class: PressedNearEvents, ns: Microsoft.MixedReality.Toolkit.Experimental.InteractiveElement,
+        asm: Microsoft.MixedReality.Toolkit.SDK.Experimental.Interactive}
```

Comment:
One downside is that the serialized info about the script (both the type and the assembly here) will change when we graduate this component. We'll need to make sure we have some migration tooling in place!